### ISSUE TITLE
pasting value into ui-select "tagged" control by "right click then paste" generates undefined value

### ISSUE THREAD

**Issue by ako977** [angular-ui/ui-select#1960]
### Bug description:
When you have a copied value in your clipboard and wish to paste it into a "multiple tagged" UI-Select, the value seems to be set as undefined IF the value was pasted with the context menu (right click menu) "paste" function.

Essentially when you paste by right clicking, then pressing the "Enter" key on your keyboard, the value is submitted as undefined and consequently becomes frozen in the ui-select control. You cannot delete it by clicking the 'remove' handle or by pressing backspace.

The error does not seem to occur on "multiple" based UI-Select controls that are not set to "tagged". 

This was observed on macOS 10.12.2: Chrome, Safari and Firefox

### Plunker
http://plnkr.co/edit/M7beWM6KfKp2BBNjET9B?p=preview

### Library Versions:
Angular 1.4.7, UI-Select 0.16.1, Bootstrap CSS

**Comment by Jefiozie** [comment]
Had a look at your issue and can you update to the latest version and verify if this issue is still present?
Unfortunately i cannot reproduce the issue as I don't have a mac.

**Comment by ako977** [comment]
@Jefiozie Yes it works now, thanks!

However just FYI for anyone else:
The bad state no longer occurs, but I didn't realize with the latest versions of UI-Select there is a new property on the directive called `paste`:
https://github.com/angular-ui/ui-select/wiki/ui-select

I was not getting any value entered when using the paste from the right clicked context menu, until I added a function name string to the `paste` attribute on the ui-select directive.
Once I defined this function and was using the value passed into it as parameter, I was able to set values via "right click pasting".
